sim/mage/talents.go

```diff
@@ -23,6 +23,7 @@ func (mage *Mage) ApplyTalents() {
 	if mage.Talents.ElementalPrecision > 0 {
 		bonusHit := 2 * float64(mage.Talents.ElementalPrecision) * core.SpellHitRatingPerHitChance
 		mage.OnSpellRegistered(func(spell *core.Spell) {
+			// TODO: This is not correct, it will affect non-class spells this way.
```

Comment:
Just FYI all of these are updated in the `mage` branch so these TODOs aren't needed 😄

Reply:
Removed them again.